builders/ethereum/verify-contracts/etherscan-plugins.md

```diff
@@ -51,9 +53,9 @@ To get started with the Hardhat Etherscan plugin, you will need to first install
 npm install --save-dev @nomicfoundation/hardhat-verify
 ```
 
-You can add your Moonscan API key to the `hardhat.config.js` file. For this example, you'll need a [Moonbeam Moonscan](https://moonscan.io){target=\_blank} API key. If you want to verify a contract on Moonriver, you'll need a [Moonriver Moonscan](https://moonriver.moonscan.io){target=\_blank} API key.
+You can add your Etherscan API key to the `hardhat.config.js` file. Because Moonscan belongs to Etherscan’s ecosystem of 50-plus chains, a single key works across all supported networks, including Moonbeam, Moonriver, and Moonbase Alpha.
```

Comment:
```suggestion
You can add your Etherscan API key to the `hardhat.config.js` file.
```

We just mentioned this on line 28, so we can just clean things up and get rid of it here.